pkg/config/types/v1alpha1/generate/controlplane.go

```diff
@@ -17,7 +17,7 @@ func controlPlaneUd(in *Input) (string, error) {
 		MachineType:     "controlplane",
 		MachineToken:    in.TrustdInfo.Token,
 		MachineCA:       in.Certs.OS,
-		MachineCertSANs: []string{},
+		MachineCertSANs: in.AdditionalSubjectAltNames,
```

Comment:
I'm not sure if `AdditionalSubjectAltNames` can be applied here for `apid` cert? Or we need a different setting in `generate.Input`?

Reply:
We have two sets of SANs in our config... we have one for k8s and one for Talos. I don't think we have a way to set the Talos SANs in the `Input` though.

Reply:
So should I introduce separate field for Talos SANs or fine to re-use?

Reply:
I think we will have to introduce another flag.

Reply:
Can't think of a better way at the moment... maybe once @rsmitty implements the ability to specify files for the user data then maybe we could generate the configs, modify them, and then create the cluster.

Reply:
Added `AdditionalMachineCertSANs` :)